### DISCUSSION TITLE
Unable to perform clustering in scp version 8

### DISCUSSION THREAD

**Discussion by aaradhyabansal** [semiautomaticgit/SemiAutomaticClassificationPlugin#342] [General]
Dear @semiautomaticgit ,

Tried several ways,but unable to perform clustering in scp version 8(the option of clustering which was previously under band processing in version 7 is no longer there in version 8).Please guide.

Sincere Regards 
Aaradhya Bansal

**Comment by semiautomaticgit** [comment]
Hello @aaradhyabansal, 
clustering is not available in version 8 (it will be added in the future).
In the meantime, you can install the previous version 7 following these steps https://semiautomaticclassificationmanual.readthedocs.io/en/latest/faq.html#can-i-use-the-previous-version-7-of-scp

**Comment by semiautomaticgit** [comment]
Hello @aaradhyabansal , clustering is now available in SCP version 8.3
